test/gui/shared/scripts/helpers/api/sharing_helper.py

```diff
@@ -130,8 +128,30 @@ def create_link_share(
         body['password'] = password
     if expire_date is not None:
         body['expireDate'] = expire_date
-    response = request.post(url, body, user=user)
+    response = request.post(get_share_url(), body, user=user)
     request.assertHttpStatus(
-        response, 200, f"Failed to create public link for resource '{resource}'"
+        response, 200, f'Failed to create public link for resource "{resource}"'
     )
     checkSuccessOcsStatus(response)
+
+
+def share_details(
+    user, share_type, uid_owner, permissions, path, item_type, share_with=None
+):
+    response = request.get(get_share_url(), user=user)
+    checkSuccessOcsStatus(response)
```

Comment:
please, rebase, PR #11892 is merged
```suggestion
    check_success_ccs_status(response)
```

Reply:
ok